storage/metrics.go

```diff
@@ -202,6 +202,34 @@ var (
 		Help: "Number of pending replicas in the split queue"}
 	metaSplitQueueProcessingNanos = metric.Metadata{Name: "queue.split.processingnanos",
 		Help: "Nanoseconds spent processing replicas in the split queue"}
+
+	// GCInfo cumulative totals.
+	metaGCGCKeys = metric.Metadata{Name: "queue.gc.info.gckeys",
+		Help: "Number of keys with GC'able data"}
+	metaGCIntentsConsidered = metric.Metadata{Name: "queue.gc.info.intentsconsidered",
+		Help: "Number of 'old' intents"}
+	metaGCIntentTxns = metric.Metadata{Name: "queue.gc.info.intenttxns",
+		Help: "Number of associated distinct transactions"}
+	metaGCTransactionSpanTotal = metric.Metadata{Name: "queue.gc.info.transactionspantotal",
```

Comment:
This doesn't really add up well in a counter (it's morally a per-Replica guage, but that doesn't make sense either). I would rename this to `TransactionSpanConsidered` (i.e. it measures how many transaction entries the GC queue had to look at, which makes sense to look at as a rate).